docs/about/contributing/getting-started-developing.md

```diff
@@ -0,0 +1,153 @@
+
+At this point, your `./config/local.yaml` should look like:
+```yaml
+---
+auth:
+    jwtPrivateKey: |
+        -----BEGIN RSA PRIVATE KEY-----
+        THISISAFAKEPRIVATEKEYTHISISAFAKEPRIVATEKEYTHISISAFAKEPRIVATEKEY
+        THISISAFAKEPRIVATEKEYTHISISAFAKEPRIVATEKEYTHISISAFAKEPRIVATEKEY
+        THISISAFAKEPRIVATEKEYTHISISAFAKEPRIVATEKEYTHISISAFAKEPRIVATEKEY
+        THISISAFAKEPRIVATEKEYTHISISAFAKEPRIVATEKEYTHISISAFAKEPRIVATEKEY
+        THISISAFAKEPRIVATEKEYTHISISAFAKEPRIVATEKEYTHISISAFAKEPRIVATEKEY
+        THISISAFAKEPRIVATEKEYTHISISAFAKEPRIVATEKEYTHISISAFAKEPRIVATEKEY
+        THISISAFAKEPRIVATEKEYTHISISAFAKEPRIVATEKEYTHISISAFAKEPRIVATEKEY
+        THISISAFAKEPRIVATEKEYTHISISAFAKEPRIVATEKEY
+        -----END RSA PRIVATE KEY-----
+    jwtPublicKey: |
+        -----BEGIN PUBLIC KEY-----
+        THISISAFAKEPUBLICKEYTHISISAFAKEPUBLICKEYTHISISAFAKEPUBLICKEY
+        THISISAFAKEPUBLICKEYTHISISAFAKEPUBLICKEYTHISISAFAKEPUBLICKEY
+        THISISAFAKEPUBLICKEYTHISISAFAKEPUBLICKEYTHISISAFAKEPUBLICKEY
+        THISISAFAKEPUBLICKEYTHISISAFAKEPUBLICKEYTHISISAFAKEPUBLICKEY
+        THISISAFAKEPUBLICKEYTHISISAFAKEPUBLICKEYTHISISAFAKEPUBLICKEY
+        THISISAFAKEPUBLICKEY
+        -----END PUBLIC KEY-----
+```
+
+#### httpd
+##### port
+This is the port to listen on. It should match wherever your API server is going to run. In this case, it'll run at http://localhost:8080, so set the port to `8080`.
+
+##### uri
+This is the externally routable URI, so set it to `http://localhost:8080`.
+
+Add this portion to your `local.yaml`:
+
+```yaml
+httpd:
+    port: 8080
+    uri: http://localhost:8080
+```
+
+#### scms
+##### github
+In this example, we'll set the github SCM. For further options, see the [default.yaml](https://github.com/screwdriver-cd/screwdriver/blob/master/config/default.yaml#L147-L185).
+
+You'll need to set the `oauthClientId` and `oauthClientSecret`. Navigate to your [Developer Settings page](https://github.com/settings/developers), click New OAuth App. Set the fields as pictured here:
+
+![Definition](../assets/scm-oauth-app.png)
+
+Click Register Application and put the Client ID and Client Secret in the fields below.
+
+```yaml
+github:
+    plugin: github
+    config:
+        oauthClientId: SOMEOAUTHCLIENTID
+        oauthClientSecret: SOMEOATHCLIENTSECRET
+        secret: RANDOMSECRETTHING
+```
+
+#### ecosystem
+##### ui
+The uri for the UI needs to be updated to your local UI, so set it to `http://localhost:4200`.
+
+```yaml
+ecosystem:
+    ui: http://localhost:4200
+```
+
+### Using your own node modules (optional)
+If you want to test out some local node modules, `npm install` them now.
+
+```bash
+$ npm install <relative_path_to_your_node_module>
+```
+
+### Starting
+Now that you've configured your `local.yaml`, save it and run:
+
+```bash
+$ npm start
+info: Server running at http://localhost:8080
+```
+or
+```bash
+$ docker run --rm -it --volume=`pwd`/local.yaml:/config/local.yaml -p 8080 screwdrivercd/screwdriver:stable
```

Comment:
`pwd` is the current directory. Is this assuming that the developer is arbitrarily making the `local.yaml` in the current directory?

The last instruction regarding the `local.yaml` that was given is to put it in `./config/local.yaml` at (currently) line 36.

Reply:
These instructions are a continuation from above, where the user should clone the repo. Added a line for them to cd into it, for people that don't understand that part.